### DISCUSSION TITLE
window focus after reload

### DISCUSSION THREAD

**Discussion by notwidow** [qtile/qtile#4032] [Q&A]
Hi, when i reload config .. then the window focus is on the last window always.. i want it to be on the window on which the mouse cursor is on.

**Comment by m-col** [comment]
Do you have `follow_mouse_focus = True` in your config?

**Reply by notwidow** [reply]
yes but id does not work when i reload. for example. i have 2 windows or 3. and if i reload. while keeping mouse on 1st. then reloading keeps my focus on last one ... not on the one with mouse. and after reload i have to click atleadt one time on window and then it will follow mouse.
